app/src/main/java/app/revanced/manager/ui/screen/PatcherScreen.kt

```diff
@@ -0,0 +1,439 @@
+                        enabled = canInstall
+                    ) {
+                        Icon(Icons.Outlined.Save, stringResource(id = R.string.save_apk))
+                    }
+                    IconButton(
+                        onClick = { vm.exportLogs(context) },
+                        enabled = patcherSucceeded != null
+                    ) {
+                        Icon(Icons.Outlined.PostAdd, stringResource(id = R.string.save_logs))
+                    }
+                },
+                floatingActionButton = {
+                    AnimatedVisibility(visible = canInstall) {
+                        ExtendedFloatingActionButton(
+                            text = {
+                                Text(
+                                    stringResource(if (vm.installedPackageName == null) R.string.install_app else R.string.open_app)
+                                )
+                            },
+                            icon = {
+                                vm.installedPackageName?.let {
+                                    Icon(
+                                        Icons.Outlined.OpenInNew,
+                                        stringResource(R.string.open_app)
+                                    )
+                                } ?: Icon(
+                                    Icons.Outlined.FileDownload,
+                                    stringResource(R.string.install_app)
+                                )
+                            },
+                            onClick = {
+                                if (vm.installedPackageName == null)
+                                    showInstallPicker = true
+                                else vm.open()
+                            }
+                        )
+                    }
+                }
+            )
+        }
+    ) { paddingValues ->
+        Column(
+            modifier = Modifier
+                .padding(paddingValues)
+                .fillMaxSize()
+        ) {
+            LinearProgressIndicator(
+                progress = {
+                    steps.flatMap { it.value }.let {
+                        it.count { step -> step.state == State.COMPLETED }.toFloat() / it.size.toFloat()
+                    }
+                },
+                modifier = Modifier.fillMaxWidth()
+            )
+            Box(
+                modifier = Modifier
+                    .verticalScroll(rememberScrollState())
+                    .fillMaxSize()
+            ) {
+                Column(
+                    verticalArrangement = Arrangement.spacedBy(16.dp),
+                    modifier = Modifier
+                        .padding(16.dp)
+                        .fillMaxSize()
+                ) {
+                    steps.forEach { (category, steps) ->
+                        Steps(category, steps)
+                    }
+                }
+            }
```

Comment:
Isn't this `Box` unnecessary?

Reply:
It is because if i apply `verticalScroll` to the Column with the padding it will add a border around the scrolling area

Reply:
Thats fair. Can you add a comment explaining it?

Reply:
Why isn't the entire screen scrollable?

Reply:
Because there's a linear progress bar fixed at the top

Reply:
> Thats fair. Can you add a comment explaining it?

Why would we need to explain it? It's just a scrollable box

Reply:
Add a comment explaining why you are using a scrollable box instead of making the column scrollable

Reply:
Solved in https://github.com/ReVanced/revanced-manager/pull/1494/commits/c5fa50354d205be92fce6207e42b5722b601ac0f